index.js

```diff
@@ -9,9 +9,20 @@ module.exports = (css, settings) => {
       `/*%%styled-jsx-expression-${id}%%*/`
     )
 
-  const preprocessed = sass({
+  // Allow a subset of node-sass options
+  const allowedOptions = ['precision', 'includePaths']
+
+  const options = Object.entries(settings).reduce((p, o) => {
```

Comment:
Not sure how well supported is Object.entries, can you use `Object.keys(settings)` instead?